lib/generators/logidze/install/check_pending.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require_relative "function_definitions"
+
+module Logidze
+  class PendingMigrationError < StandardError
```

Comment:
I would love to make it actionable (and allow user to upgrade in a single click). Do you think that's possible? Looks like we need to call `rails g logidze:install --update` and `rails db:migrate`. At least for the cases when `logidze_logger` hasn't changed (and thus, triggers shouldn't be recreated) that seems feasible.

See https://github.com/rails/rails/blob/098fd7f9b3d5c6f540911bc0c17207d6b48d5bb3/activerecord/lib/active_record/migration.rb#L136-L144